packages/db/src/resources/bytecodes.ts

```diff
@@ -20,7 +20,7 @@ export const bytecodes: Definition<"bytecodes"> = {
   typeDefs: gql`
     type Bytecode implements Resource {
       bytes: Bytes!
-      linkReferences: [LinkReference]
+      linkReferences: [LinkReference!]
```

Comment:
This should be allowed to be empty, shouldn't it? Empty array if the bytecode has no link references? Maybe I missed something...

Reply:
See above for how we were wrong about what this means; it doesn't mean at least one item, it means the array can't have any null values in it. Which is what we want, I'd say - we won't get missing link references ever (like we will for `Compilation.sources`, e.g.)